### PR TITLE
Update documentation and requirements

### PR DESCRIPTION
**Pull request by vogelpi**
This PR contains two commits to:
- update the README.md and more importantly the getting started guide, and
- update the Python requirements.

### CODE REVIEW

**Review by moidx — APPROVED** [review]
LGTM. Nice doc!

File: doc/getting_started.md [hunk 1]
```diff
@@ -1,53 +1,208 @@
...
 
-## Running capture and attack
+To this end, follow these steps:
 
-Sample capture run:
+1. Go to the root directory of the OpenTitan repository.
+1. Before generating the OpenTitan FPGA bitstream for the CW305 target board,
+   you first have to run
+```console
+$ ./hw/top_earlgrey/util/opentitan_earlgrey_reduce_flash_size.py
```
Comment (moidx) [inline doc/getting_started.md:65]: TODO: Add details on how to update the AES IP parameters for testing.

Comment (vogelpi) [reply doc/getting_started.md:65]: Note, this is no longer required. The corresponding parameters are now propagated up to the top level and driven by the CW305-specific FPGA top ;-)

**Comment by vogelpi** [comment]
Thanks @moidx , I am merging this now.

**Pull request merged**